src/maker/rpc/messages.rs

```diff
@@ -56,6 +50,30 @@ pub enum RpcMsgReq {
     SyncWallet,
 }
 
+/// Represents balance of each utxo type.
+pub struct Balance {
+    /// Seed balance.
+    pub regular: Amount,
+    /// Incoming swap balance.
+    pub swap: Amount,
+    /// Unfinished timelock contract balance.
+    pub contract: Amount,
+    /// Amount locked in Fidelity bonds.
+    pub fidelity: Amount,
+    /// Spendable amount in wallet (seed + swap balance).
+    pub spendable: Amount,
+}
+
+impl Display for Balance {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "Seed balance: {}, Swap balance: {}, Contract balance: {}, Fidelity balance: {}, Spendable balance: {}",
+            self.regular.to_sat(), self.swap.to_sat(), self.contract.to_sat(), self.fidelity.to_sat(), self.regular.to_sat() + self.swap.to_sat()
+        )
+    }
+}
```

Comment:
Can we have a more prettier json like display instead of in one line? checkout bitcoin core's `getbalances` rpc output.

Reply:
Instead of display implementation, we should just format the output at the app main function. Also the display needs to be different for takers and makers. For takers there is no fidelity balance.